app/javascript/includes/save_post.js

```diff
@@ -1,11 +1,11 @@
 $( document ).on('turbolinks:load', function() {
-  $(".save").on("click", function(){
-    let post_id = $(this).data("id");
+  $(".saved").on("click", '.save', function(){
+    let post_id = $(this).parent().data("id");
 
     $.ajax({
-      url: "p/"+post_id+"/save",
-      method: "PATCH",
-      data: {},
+      url: "save_post?post_id="+post_id,
```

Comment:
why don't you use a clean url like the following:
```
/save-post/<post-id>
```

Reply:
url changed as '/save_post/post_id'